plugin-api/src/main/java/net/clgd/ccemux/api/config/package-info.java

```diff
@@ -0,0 +1,10 @@
+/**
+ * This package contains the api for the modular configuration system used by
+ * CCEmuX. Any type can be stored, although "basic" types (boxed primitives,
+ * strings, lists, etc) should be preferred for compatibility reasons.
+ * 
+ * @see Config
+ * @see ConfigProperty
+ * @see Group
+ */
+package net.clgd.ccemux.api.config;
```

Comment:
It might be worth installing [an editor config](http://editorconfig.org/#download) plugin for whatever editor you use. Missing trailing newlines make me sad.

Reply:
It's set to insert newlines, I think I just hadn't told it to format, derp.